src/core/api/playback_observer.ts

```diff
@@ -1288,9 +1288,11 @@ function getInitialObservation(mediaElement: HTMLMediaElement) : IPlaybackObserv
   const mediaTimings = getMediaInfos(mediaElement);
   return objectAssign(mediaTimings,
                       { rebuffering: null,
-                        event: "init",
+                        event: "init" as const,
                         seeking: SeekingState.None,
-                        position: { last: mediaTimings.position,
-                                    pending: undefined },
-                        freezing: null });
+                        position: new ObservationPosition(mediaTimings.position, null),
```

Comment:
:O

Reply:
is it correct to correct it like that ?

Reply:
It is, I was surprised because the previous line seemed like an important issue, though the position of the initial observation may have never been actually used...

It will be merged for `-rc1` but not for our already scheduled-to-be-tested and hopefully future in-production pre-release from yesterday I guess.